package.json

```diff
@@ -35,7 +35,7 @@
     "prettier": "^2.0.2"
   },
   "scripts": {
-    "test": "c8 --reporter=lcov --reporter=text mocha",
+    "test": "c8 --reporter=lcov --reporter=text mocha --timeout 15000",
```

Comment:
I can switch the tests to use a fake timer implementation if this is distasteful -- wasn't sure what would be preferred